timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -351,19 +350,17 @@ void declare_context(py::module &m) {
             "multiple_steps",
             [](Context &ctxt, const int n_steps, int store_x_interval) -> py::tuple {
                 // (ytz): I hate C++
-                int x_interval = (store_x_interval <= 0) ? n_steps : store_x_interval;
-                std::array<std::vector<double>, 2> result = ctxt.multiple_steps(n_steps, x_interval);
-
                 int N = ctxt.num_atoms();
                 int D = 3;
-                int F = result[0].size() / (N * D);
-                py::array_t<double, py::array::c_style> out_x_buffer({F, N, D});
-                std::memcpy(out_x_buffer.mutable_data(), result[0].data(), result[0].size() * sizeof(double));
-
-                py::array_t<double, py::array::c_style> box_buffer({F, D, D});
-                std::memcpy(box_buffer.mutable_data(), result[1].data(), result[1].size() * sizeof(double));
 
-                return py::make_tuple(out_x_buffer, box_buffer);
+                int x_interval = (store_x_interval <= 0) ? n_steps : store_x_interval;
```

Comment:
fwiw, if `store_x_interval < 0` we should probably just assert

Reply:
It asserts within the function, which tends to be our convention though it is not consistent: https://github.com/proteneer/timemachine/blob/8e46cf807cc922ae4f3ab1ee39f11d1ff6c4970c/timemachine/cpp/src/context.cu#L252-L254

Looks like this was not previously tested, will add a test.